packages/strapi-admin/admin/src/components/Roles/Permissions/ContentTypes/ContentTypesRow/index.js

```diff
@@ -53,16 +51,18 @@ const ContentTypeRow = ({ index, contentType, permissionsLayout }) => {
   }, [contentType, contentTypesPermissions]);
 
   const actionsForConditions = useMemo(() => {
-    return contentTypeActions.map(action => ({
+    const existingActions = getAllAttributesActions(contentType.uid, contentTypesPermissions);
```

Comment:
Can you declare a constant so you don't have to compute the getAllAttributesAction twice ?